unfetter-discover/unfetter-store
# updates for adding stage

Requires the unfetter project `https://github.com/unfetter-discover/unfetter/pull/908`
fixes unfetter-discover/unfetter#855

## To Test
* verify travis passed w/ 
```
cd unfetter-store/unfetter-ctf-ingest
npm run build && npm run lint && npm run test
```
